swarm/network/pss.go

```diff
@@ -436,13 +470,24 @@ func (self *PssProtocol) handle(msg []byte, p *p2p.Peer, senderAddr []byte) erro
 	return nil
 }
 
-func (ps *Pss) isSelfRecipient(msg *PssMsg) bool {
-	if bytes.Equal(msg.GetRecipient(), ps.Overlay.GetAddr().OverlayAddr()) {
+func (self *Pss) IsSelfRecipient(msg *PssMsg) bool {
+	if bytes.Equal(msg.GetRecipient(), self.Overlay.GetAddr().OverlayAddr()) {
 		return true
 	}
 	return false
 }
 
+func (self *Pss) GetPingHandler() func([]byte, *p2p.Peer, []byte) error {
```

Comment:
It's used by swarm/swarm.go, convenience test for implementers, will be removed when we're ready for productions. It's ok no?